cross-doc-explainer.md

```diff
@@ -140,16 +138,62 @@ customizable as same-document transitions, while allowing good defaults that wor
 Same document transitions can be programatically extended (until the [updateCallbackDone](https://drafts.csswg.org/css-view-transitions-1/#dom-viewtransition-updatecallbackdone) promise is fullfilled), [skipped](https://drafts.csswg.org/css-view-transitions-1/#dom-viewtransition-skiptransition), programatically animated using the [ready](https://drafts.csswg.org/css-view-transitions-1/#dom-viewtransition-ready) promise, and their [end time can be
 observed](https://drafts.csswg.org/css-view-transitions-1/#dom-viewtransition-finished).
 
-To accomplish the same control and observability, the developer would need access to a
-[ViewTransition](https://drafts.csswg.org/css-view-transitions-1/#the-domtransition-interface)
-object. To achieve that, we propose to fire a `reveal` event at both lifecycle moments, with a `ViewTransition` object.
-See [whatwg/html#9315](https://github.com/whatwg/html/issues/9315), w3c/csswg-drafts#8682, and w3c/csswg-drafts#8805.
+To achieve the same level of control for cross document navigations we propose two new events, corresponding to last pre-snapshot moment in the old and new documents.
+
+### `pageconceal`
+
+whatwg/html#9702
+
+To allow customizing a view transition from the old document, fire a `pageconceal` event just before capturing snapshots on the outgoing document. This allows the author
+to customize or skip a view transition based on the final destination URL:
+
+Note: `event.activation` is a `NavigationActivation`, see the [NavigationActivation explainer](https://github.com/WICG/view-transitions/blob/main/navigation-activation-explainer.md).
+
+```js
+window.addEventListener("pageconceal", event => {
+  if (!event.viewTransition)
+    return;
+  const path = new URL(event.activation.entry).pathname;
+  if (path === '/logout') {
+    // No transition to logout page.
+    event.viewTransition.skipTransition();
+  } else if (path === '/home') {
+    // Don't capture navigation-bar animation when navigating to home.
+    navigationBar.viewTransitionName = "none";
+  }
+});
+```
+
+Reusing the `navigate` or `pagehide` events was considered. `navigate` doesn't work because it doesn't provide the final URL (after redirects), doesn't fire for all navigation types, and fires before the page would be frozen for snapshot capture. `pagehide` doesn't work because the capture needs to asynchronously update the rendering which shouldn't happen after `pagehide`.
```

Comment:
A lot of this is still in consideration. Not sure if we should try to capture that in the explainer.

Reply:
We had `reveal` here while it was still under heavy consideration so I figured this made sense too. I'd prefer to leave it in, with an added disclaimer that it's still under consideration, as it's a better representation of what we're considering. But if you disagree I'm also happy to just remove it wholesale for now.

Reply:
On second thought (based on our VT sync), I removed it and left just the link to the CSSWG issue where we're discussing this.